specification/gedcom-01-serialization.md

```diff
@@ -403,7 +403,7 @@ defines the following tags
 | `_MEMBER` | `http://xmlns.com/foaf/0.1/member` |
 :::
 
-The meaning of a documented extension tag is identified by its URI, not its tag.
+The meaning of a documented extension tag is identified by its superstructure type and URI, not its tag.
```

Comment:
```suggestion
The meaning of a documented extension tag is identified by its superstructure type and its URI, not its tag.
```
to avoid misreading it as the URI of the superstructure